cfme/common/vm.py

```diff
@@ -568,7 +568,7 @@ class VM(BaseVM):
 
     retire_form = Form(fields=[
         ('date_retire',
-            AngularCalendarInput("retirement_date",
+            AngularCalendarInput("retirement_date_datepicker",
```

Comment:
**required** is this a bugfix or does it need a version pick?

Reply:
Actually, you are right, this needs version pick